### PR TITLE
Feature/child trophy room responsive design

### PR DESCRIPTION
**Pull request by Robert-Lark**
# Description
RESPONSIVE-DESIGN-ADJUSTMENTS
This pull request relates to design changes on the child/trophyroom page. The changes allow the page to be viewed at tablet and mobile breakpoints without formatting issues. 

Fixes #

- What work was done?
added responsive breakpoint css styling for tablet and mobile viewports. 

- Why was this work done?
to ensure smooth UI flow throughout the application on all viewports. 

- What feature / user story is it for?
As a user I want to be able to View app on multiple devices/browsers to ensure UI is responsive.

## Change Status

- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

## Has This Been Tested

- [ ] No

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

File: src/components/pages/TrophyRoom/RenderTrophyRoom.js [hunk 1]
```diff
@@ -49,8 +49,12 @@ const RenderTrophyRoom = props => {
   return (
     <>
       <Header displayMenu={true} title="Trophy Room" />
-      <Button style={{ margin: '1rem' }} onClick={dashboard}>
-        Back to Child Dashboard!
```
Comment (Robert-Lark) [inline src/components/pages/TrophyRoom/RenderTrophyRoom.js]: added and removed "!" so I could make a draft pull request

File: src/styles/less/TrophyRoom.less [hunk 2]
```diff
@@ -1,15 +1,45 @@
 @import '../less/variables.less';
```
Comment (Robert-Lark) [inline src/styles/less/TrophyRoom.less:2]: TrophyRoom.less is the main file I worked in - it contains the CSS code for the trophyroom and leaderboard table.

**Comment by Robert-Lark** [comment]
Review is again required as there was a merge conflict in the package-lock.json file. I think what happened - I may be wrong - is when we cloned the repo and ran NPM install - some of the packages updated to newer versions. Package-lock.json is automatically generated for any operations where npm modifies either the node_modules tree, or package.json.  I accepted both the incoming changes and the original file and the conflict was resolved. I imagine this will be an issue everyone will face on their first merge.

**Review by leachtucker — APPROVED** [review]
Looks good. And nice work updating those less variables for media queries. I will take advantage of those in my future code on this project.

**Review by cearc-sara — APPROVED** [review]
This looks great, great job with the responsive design

**Pull request merged**